### DISCUSSION TITLE
Openwrt 23.05.2 ?? Questions and more.....

### DISCUSSION THREAD

**Discussion by Ninotaire** [xchwarze/wifi-pineapple-cloner#16] [Q&A] [answered]
Hello, 
This last openwrt version, is compatible with your tool?
Does it only work on the routers on your list?......That is, it only works on mipsel chipset?

**Comment by xchwarze** [comment, marked as answer]
Exactly. It is all explained in build.md. Anyway if you want something more modern you can read this explanation https://github.com/xchwarze/wifi-pineapple-cloner/discussions/17

**Reply by Ninotaire** [reply]
Thanks xchwarze.
